src/cockpit/389-console/src/lib/replication/replModals.jsx

```diff
@@ -1593,18 +1593,19 @@ export class AddManagerModal extends React.Component {
             manager,
             manager_passwd,
             manager_passwd_confirm,
-            error
+            error,
+            edit,
         } = this.props;
-        let saveBtnName = _("Add Replication Manager");
+        let saveBtnName = this.props.edit ? "Save Replication Manager" : _("Add Replication Manager");
         const extraPrimaryProps = {};
         if (spinning) {
-            saveBtnName = _("Adding Replication Manager ...");
+            saveBtnName = this.props.edit ? "Saving replication Manager ..." : _("Adding Replication Manager ...");
```

Comment:
`replication`, probably, should be capitalized too:)

Reply:
> `replication`, probably, should be capitalized too:)

Yup...